data/json/mapgen/Prepper_house01.json

```diff
@@ -127,10 +102,9 @@
             ],
             "place_monsters": [
                 { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "repeat": [2,3] },
-                { "monster": "GROUP_PREPPER_HOUSE", "x": [ 2, 21 ], "y": [ 13, 21 ] }
-            ],
-            "comment": "end of mapgen json definition ---v"
-        },
-        "comment": "end of mapgen entry ---v"
+                { "monster": "GROUP_PREPPER_HOUSE", "x": [ 2, 21 ], "y": [ 13, 21 ] },
+                { "monster": "GROUP_PEST", "x": [ 2, 21 ], "y": [ 13, 21 ], "CHANCE": 3 }
```

Comment:
This monstergroup doesn't exist in core. Core mapgen should not contain mod-only info.
Black rats would fit here.